lib/section-to-webhook.ts

```diff
@@ -51,17 +52,40 @@ const getDescription = ($: cheerio.Root): string =>
     .map((html) => turndownService.turndown(html))
     .join("\n\n");
 
-const getActions = ($: cheerio.Root): string[] => {
-  const unwantedActions = ["true", "false", "status"];
-
+const getPropertiesEl = ($: cheerio.Root) => {
   const $table = $('[id^="webhook-payload-object"]').next("table");
 
   if (!$table.is("table")) {
     return [];
   }
 
-  const [keyEl, , descriptionEl] = $table
-    .find("tbody tr:first-child td")
+  return $table.find("tbody tr").get() as cheerio.Element[][];
+};
+
+const getProperties = ($: cheerio.Root) => {
```

Comment:
```suggestion
const getProperties = ($: cheerio.Root): Webhook['properties']  => {
```

Would be good to have the return type here so that TS ensures the implementation is returning the correct type (otherwise TS will error on line 38 if the implementation returns something different, which implies the issue is on that line rather than with whats being returned).

(ideally the other methods should have return types too)